github/Organization.py

```diff
@@ -662,6 +664,9 @@ def create_team(
         assert is_optional(permission, str), permission
         assert is_optional(privacy, str), privacy
         assert is_optional(description, str), description
+        assert notification_setting is NotSet or isinstance(notification_setting, str), notification_setting
+        if notification_setting is not NotSet:
+            assert notification_setting in ["notifications_enabled", "notifications_disabled"]
```

Comment:
This can be simplified:
```suggestion
        assert notification_setting in ["notifications_enabled", "notifications_disabled", NotSet], notification_setting
```

Reply:
Just followed an existing pattern from the file, but this way is better for sure.